statsd/statsd.go

```diff
@@ -278,6 +286,23 @@ func newWithWriter(w statsdWriter, o *Options, writerName string) (*Client, erro
 		c.Tags = append(c.Tags, entityTag)
 	}
 
+	// Inject DD_ENV, DD_SERVICE, and DD_VERSION as global tags if present.
+	env := os.Getenv(ddEnvName)
+	if env != "" {
+		envTag := fmt.Sprintf("%s:%s", ddEnvTagName, env)
```

Comment:
We could make this generic to ease adding support to more env tags.

something like:
```go
var envTagsMapping := map[string]string{
  "DD_ENV":     "env".
  "DD_SERVICE": "service",
  "DD_VERSION": "version",
}

// [...]

for envName, tagName := range envTagsMapping {
    if value := os.Getenv(envName); value != "" {
        c.Tags = append(c.Tags, fmt.Sprintf("%s:%s", tagName, value))
    }
}
```

WDYT ?

Reply:
This is a good idea, I will make the change!